### DISCUSSION TITLE
When I enter my JCB card, it is recognized by DISCOVER.

### DISCUSSION THREAD

**Discussion by koki-fujiwara** [community/community#138870] [Enterprise]
### Select Topic Area

Question

### Body

To sign up for the Team plan, I entered my JCB brand card information. Is it possible to make a payment with a JCB card?

**Comment by queenofcorgis** [comment]
Hi there,

Was your card declined? If so, please review [the doc](https://docs.github.com/en/billing/managing-your-github-billing-settings/troubleshooting-a-declined-credit-card-charge) on troubleshooting to resolve ❤️ .
